subprojects/gradle/gradle-logger/src/main/kotlin/com/avito/logger/GradleLoggerFactory.kt

```diff
@@ -0,0 +1,64 @@
+package com.avito.logger
+
+import com.avito.android.elastic.ElasticConfig
+import com.avito.android.sentry.SentryConfig
+import com.avito.android.sentry.sentryConfig
+import com.avito.utils.gradle.BuildEnvironment
+import com.avito.utils.gradle.buildEnvironment
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.Task
+import java.io.Serializable
+
+class GradleLoggerFactory(
+    private val isCiRun: Boolean,
+    private val sentryConfig: SentryConfig,
+    private val elasticConfig: ElasticConfig,
+    private val projectPath: String,
+    private val pluginName: String? = null,
+    private val taskName: String? = null
```

Comment:
Unused

Reply:
thanks! IDE didn't help me with it
fixed